app/src/main/res/layout/sign_up_fragment.xml

```diff
@@ -11,113 +11,117 @@
             type="org.fossasia.openevent.app.common.data.models.User" />
     </data>
 
-    <android.support.v4.widget.NestedScrollView
+    <FrameLayout
```

Comment:
@iamareebjamal using data binding to set the visibility of the included view directly is not working. Will have to go with this. Same is used for the empty layout too.